docs/source/operations/projections/som.rst

```diff
@@ -0,0 +1,71 @@
+.. _som:
+
+********************************************************************************
+Space Oblique Mercator (SOM)
+********************************************************************************
+
++---------------------+----------------------------------------------------------+
+| **Classification**  | Conformal                                                |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forward and inverse, spherical and ellipsoidal           |
++---------------------+----------------------------------------------------------+
+| **Defined area**    | Global                                                   |
++---------------------+----------------------------------------------------------+
+| **Alias**           | som                                                      |
++---------------------+----------------------------------------------------------+
+| **Domain**          | 2D                                                       |
++---------------------+----------------------------------------------------------+
+| **Input type**      | Geodetic coordinates                                     |
++---------------------+----------------------------------------------------------+
+| **Output type**     | Projected coordinates                                    |
++---------------------+----------------------------------------------------------+
+
```

Comment:
It would be great to have some background info here (see https://proj.org/operations/projections/merc.html for a good example). You seem to know what this projection is used for and possibly also the history behind it. It would be great to capture that knowledge here so the next person doesn't start from scratch.

I guess this also goes for the misrsom projection.

Reply:
I've given a shot at a bit of background and linked to the relevant references which hopefully should cover the curious.

Reply:
Looks great, thanks!